server/cli/init/base-file-sources/hooks.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
A few things here..

* Order the imports alphabetically, easier to read.
* Import bluebird here and promisify fs